src/controllers/crate_owner_invitation.rs

```diff
@@ -47,6 +49,7 @@ pub fn list(req: &mut dyn RequestExt) -> EndpointResult {
     Ok(req.json(&json!({
         "crate_owner_invitations": crate_owner_invitations,
         "users": users,
+        "meta": meta,
```

Comment:
Adds `meta` to the response similar to how its done for `crates` endpoint, this way we can retrieve the seek hash for the next page.

Reply:
sorry that I just noticed this now, but https://github.com/rust-lang/crates.io/pull/3763 introduced a new endpoint for the invitations (`GET /api/private/crate_owner_invitations`) that already supports seek-based pagination. I guess it would be best if we first switched to this new endpoint in the frontend in a dedicated MR and then implemented pagination on top of it. let me know if you want to tackle this :)

Reply:
Sounds good! Lets do that!
So we close this PR?

Reply:
whatever you prefer. feel free to keep it open and then rebase it once the other stuff is done :)